examples/ui.content/src/main/content/jcr_root/content/core-components-examples/library/adaptive-form/titlev2/.content.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This should not be required, the existing title component should point to v2

Reply:
Removed!